editors/code/src/spIndex.ts

```diff
@@ -122,8 +123,19 @@ export async function activate(context: vscode.ExtensionContext) {
   );
 
   context.subscriptions.push(
-    vscode.window.onDidChangeActiveTextEditor((event) => {
-      let folder = vscode.workspace.getWorkspaceFolder(event.document.uri);
+    vscode.window.onDidChangeActiveTextEditor((editor) => {
+      // We check for a valid path in the editor's filename,
+      // which would indicate we're not on an output console
+      if (!path.isAbsolute(editor.document.fileName)) {
```

Comment:
I think a better way to check for this is through the scheme of the document's uri?

Reply:
Hm can you illustrate a bit? What exactly would you check? Parse the filename as URI and extract stuff or something?

Reply:
```suggestion
      if (editor.document.uri.scheme === "output") {
```
More generally, the scheme of a URI is a specification that allows to assign reserved characters to the rest of the URI.
In our case, the relevant part is that a document's URI scheme can be a `file` or `output` (and others), `output` being for the output console.

Reply:
Cool!